share/sysbuild/images/bootloader/Kconfig

```diff
@@ -30,6 +30,87 @@ endchoice
 
 if BOOTLOADER_MCUBOOT
 
+choice MCUBOOT_MODE
+	prompt "Mode of operation"
+	default MCUBOOT_MODE_SWAP_WITHOUT_SCRATCH
+	help
+	  The operating mode of MCUboot (which will also be propagated to the application).
+
+config MCUBOOT_MODE_SINGLE_APP
```

Comment:
Does this somehow correspond to https://github.com/zephyrproject-rtos/zephyr/blob/main/modules/Kconfig.mcuboot ? Do we have to have both files?

Reply:
Yes, these are sysbuild Kconfig entries, which will get set in mcuboot and the application